### PR TITLE
Translate README to Russian

### PR DESCRIPTION
**Pull request by unger1984**
# Description

Create README-RU.md and link to it from other README.

- New feature (non-breaking change which adds functionality)

# Checklist:

If something is unclear, please submit the PR anyways and ask about what you thought was unclear.

- [x] This branch is based on `v1.0.0`
- [x] This PR is targeted to merge into `v1.0.0` (not `master` or `develop`)
- [x] I have added an entry under `[next]` in `CHANGELOG.md`
- [x] I have formatted my code with `flutter format`
- [x] I have made corresponding changes to the documentation
- [ ] I have added examples for new features in `doc/examples`
- [x] The continuous integration (CI) is passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The current v1 release is `1.0.0-rc2` on pub. The last stable version so far is
 
 ---
 
-[English](https://github.com/flame-engine/flame) | [简体中文](README-ZH.md) | [Polski](https://github.com/mbiesiad/flame/blob/pl_PL/README-PL.md)
+[English](https://github.com/flame-engine/flame) | [简体中文](README-ZH.md) | [Polski](https://github.com/mbiesiad/flame/blob/pl_PL/README-PL.md) | [Русский](README-RU.md)
```
Comment (spydon) [inline README.md:17]: Not a new introduction, but this shouldn't point to a separate repository

File: README-RU.md [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+Мы также предлагаем тщательно подобранный список игр, библиотек и статей на сайте [awesome-flame](https://github.com/flame-engine/awesome-flame).
+
+Обратите внимание что некоторые статьи могут быть устаревшими, но все же они довольно полезны.
+
+## Благодарности
+
+ * [Fireslime](https://fireslime.xyz), команда ответственная за поддержку Flame.
+ * Все дружелюбные участники и люди которые помогают в сообществе.
+ * [Luanpotter](https://github.com/luanpotter)'s основатель Flame и [audioplayers](https://github.com/luanpotter/audioplayer) библиотеки, которая является форком [rxlabz's](https://github.com/rxlabz/audioplayer).
+ * Дарт порт [Box2D](https://github.com/google/box2d.dart).
```
Comment (spydon) [inline README-RU.md:113]: This is outdated, it is now Forge2D

File: README.md (hunk 1 above)
Comment (unger1984) [reply README.md:17]: In English make relative link too?

Comment (spydon) [reply README.md:17]: Exactly

File: README-RU.md [hunk 3]
```diff
@@ -0,0 +1,113 @@
+[![Pub](https://img.shields.io/pub/v/flame.svg?style=popout)](https://pub.dartlang.org/packages/flame) ![Test](https://github.com/flame-engine/flame/workflows/Test/badge.svg?branch=master&event=push) [![Discord](https://img.shields.io/discord/509714518008528896.svg)](https://discord.gg/pxrBmy4)
+
+<img src="https://i.imgur.com/vFDilXT.png" width="400">
+
+## О версии 1.0.0
+
+Это релиз-кандидат версии 1.0.0 движка Flame. Эта версия не готова к использованию в продакшене, часть когда из документации не адаптирована под нее, кроме того, многое еще может поменяться перед релизом.
+
+Используйте эту версию если хотите посмотреть каким будет Flutter и оставить обратную связь по структуре или предложить новые фичи.
```
Comment (spydon) [inline README-RU.md:9]: Shouldn't this say "Flame" and not "Flutter"?

Comment (unger1984) [reply README-RU.md]: sure!

**Review by spydon — APPROVED** [review]

**Review by erickzanardo — APPROVED** [review]
LGTM

**Pull request merged**